src/ui/colorui.js

```diff
@@ -18,6 +18,10 @@ import {
 /**
  * The color UI plugin which isolates the common logic responsible for displaying dropdowns with color grids.
  *
+ * In case that `documentColors` for given plugins are not defined in the configuration,
+ * then by default its value equals to its `column` value. This results with
+ * displaying one row of document colors.
```

Comment:
This paragraph is irrelevant to the `ColorUI` class. It should be (and is) documented in the config property.